src/bigint.rs

```diff
@@ -366,6 +366,16 @@ impl Num for BigUint {
     /// Creates and initializes a `BigUint`.
     fn from_str_radix(s: &str, radix: u32) -> Result<BigUint, ParseBigIntError> {
         assert!(2 <= radix && radix <= 36, "The radix must be within 2...36");
+        let mut s = s;
+        if s.starts_with('+') {
+            s = &s[1..];
+            if s.starts_with('+') {
+                // create ParseIntError::InvalidDigit
```

Comment:
Must you handle a second '+' specifically though?  Seems if you just let it continue, that would get flagged invalid in the byte-conversion step.  I guess you avoid creating that Vec, but there's no need to optimize for error cases.

You should add a "++" test either way, since this is a case you thought about.